backend/experiment/actions/playback.py

```diff
@@ -17,34 +19,48 @@
 PLAY_BUFFER = "BUFFER"
 PLAY_NOAUDIO = "NOAUDIO"
 
+PlayMethods = Literal["EXTERNAL", "HTML", "BUFFER", "NOAUDIO"]
+
+
+class PlaybackSection(TypedDict):
+    id: int
+    url: str
+    group: str
```

Comment:
NB: `group` is not needed anymore. This was included at some point to assist the `MatchingPairs` evaluation when it was still done in the frontend.

Reply:
I've added a TODO to the code, or do you think it's as easy as just removing it?